Update default-probing.md (#50134)
dotnet/docs: docs/core/dependency-loading/default-probing.md

Additionally, the *\*.deps.json* files for any referenced frameworks are similarly parsed.

The environment variable `DOTNET_ADDITIONAL_DEPS` can be used to add additional dependencies.  `dotnet.exe` also contains an optional `--additional-deps` parameter to set this value on application startup.
> [!NOTE]
> The `DOTNET_ADDITIONAL_DEPS` environment variable and the `--additional-deps` command-line option are **ignored for self-contained applications**.  

> > These options only apply to **framework-dependent applications**.  

Reviewer: ```suggestion
> These options are **ignored for self-contained applications**. 
```
